doc/UserGuide.md

```diff
@@ -42,11 +42,13 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Put a `p` before the phone / email / address prefixes to mark it as `private`. `private` details can only
 > be seen using the `viewall` command.
 > 
+> A valid address is entered in the following format a/BLOCK, STREET, UNIT, POSTAL_CODE
+>
 > Persons can have any number of tags (including 0)
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/criminal t/friend`
+* `add John Doe p/98765432 e/johnd@gmail.com a/block 123, John street, #01-01, 111111`
```

Comment:
Well done on updating the user guide! However, this is considered as a significant change and most importantly, you wouldn't want to commit your user guide together with your actual java implementations as it confuses the evaluator ~ do try to take note of this in the future :)